lib/std/os/uefi.zig

```diff
@@ -23,9 +23,51 @@ pub var handle: Handle = undefined;
 /// A pointer to the EFI System Table that is passed to the EFI image's entry point.
 pub var system_table: *tables.SystemTable = undefined;
 
+/// UEFI's memory interfaces exclusively act on 4096-byte pages.
+pub const Page = [4096]u8;
+
 /// A handle to an event structure.
 pub const Event = *opaque {};
 
+pub const EventRegistration = *const anyopaque;
```

Comment:
An event registration is not arbitrary data (as opposed to say, a notify context which is user-controlled), it refers to a specific opaque structure
```suggestion
pub const EventRegistration = *const opaque{};
```